src/System.Private.CoreLib/src/System/Runtime/RuntimeImportAttribute.cs

```diff
@@ -8,8 +8,8 @@ namespace System.Runtime
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor, Inherited = false)]
     public sealed class RuntimeImportAttribute : Attribute
     {
-        public string DllName;
-        public string EntryPoint;
+        private string DllName;
```

Comment:
To follow design guidelines for defining attributes, these should be read-only properties, like this: https://github.com/dotnet/corert/blob/229bc90dc82c28d0c1a259b07995628391eb86f9/src/System.Private.CoreLib/shared/System/Runtime/InteropServices/DllImportAttribute.cs#L15